fixtures/README.md

```diff
@@ -0,0 +1,14 @@
+# Browser Fixtures
+
+This folder contains tools to assist in the testing of browser quirks.
+
+Each document should be browsable without setting up a static file server. Simply
+open the test case in a browser.
+
+## Browser Targets
+
+IE9+
+IE Edge
+Chrome
+Firefox
+Safari
```

Comment:
Curiously, I couldn't find anything about base line browser support for React. Would it be the same as Facebook? Either way, do you know what this support matrix looks like?

Otherwise, drawing from the list of ES5 compliant browsers makes the most sense to me:
http://caniuse.com/#search=ES5